xml/common_intro_feedback_i.xml

```diff
@@ -16,66 +16,67 @@
  </info>
 
  <para>
-  Several feedback channels are available:
+  Your feedback and contribution to this documentation is welcome!
+  Several channels are available:
  </para>
 
  <variablelist>
   <varlistentry os="sles;sled">
-   <term>Bugs and Enhancement Requests</term>
+   <term>Service Requests and Support</term>
    <listitem>
-    <para os="sles;sled">
+    <para>
      For services and support options available for your product, refer to
      <link xlink:href="http://www.suse.com/support/"/>.
     </para>
     <para>
-     Help for &opensuse; is provided by the community. Refer to <link
-     xlink:href="https://en.opensuse.org/Portal:Support"/> for more information.
-    </para>
-    <para os="sles;sled">
-<!--taroth 2015-06-01: http://bugzilla.suse.com/show_bug.cgi?id=882059-->
-     To report bugs for a product component, go to
-     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and
-     click <guimenu>Create New</guimenu>.
-    </para>
-    <para os="osuse">
-     To report bugs &productname;, go to <link
-     xlink:href="https://bugzilla.opensuse.org/"/>, log in, and click
-     <guimenu>New</guimenu>.
+     To open a service request, you need a subscription at &scc;. Go to
+     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and click
+     <guimenu>Create New</guimenu>.
     </para>
    </listitem>
   </varlistentry>
-  <varlistentry os="osuse">
+  <varlistentry>
    <term>Bug Reports</term>
    <listitem>
     <para>
-     To report bugs for &productname;, go to
-     <link xlink:href="https://bugzilla.opensuse.org/"/>, log in, and
-     click <guimenu>New</guimenu>.
+     To report documentation bugs in
+     <link os="sles;sled" xlink:href="https://bugzilla.suse.com/"/><link os="osuse" xlink:href="https://bugzilla.opensuse.org/"/>,
```

Comment:
The wording "report bugs in ..."  is appropriate when used like "report bugs in $product" but "report bugs in $url" is a bit weird. Suggestion: `Report issues with the documentation at bugzilla.suse.com. To simplify this process, you can use the "Report Documentation Bug" links next to document headlines. These preselect the right product and category and add a link to the current section.`

Reply:
@sknorr : 100 points - I was wondering about the preposition, too. :) I mostly went with your proposal but re-added the HTML version again (because this function is not available in PDF nor EPUB).